src/Avalonia.Base/Input/GestureRecognizers/HoldGestureRecognizer.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System.Timers;
+using Avalonia.Input.GestureRecognizers;
+using Avalonia.Threading;
+
+namespace Avalonia.Input
+{
+    public class HoldGestureRecognizer : StyledElement, IGestureRecognizer
+    {
+        private const int Tolerance = 30;
+        private IInputElement? _target;
+        private IGestureRecognizerActionsDispatcher? _actions;
+        private int _gestureId;
+        private IPointer? _tracking;
+        private PointerPressedEventArgs? _pointerEventArgs;
+        private Rect _trackingBounds;
+        private Timer? _holdTimer;
+        private bool _elasped;
+
+        /// <summary>
+        /// Defines the <see cref="IsHoldWithMouseEnabled"/> property.
+        /// </summary>
+        public static readonly StyledProperty<bool> IsHoldWithMouseEnabledProperty =
```

Comment:
How to set this property from XAML?

Reply:
```
        <Control.GestureRecognizers>
          <HoldGestureRecognizer IsHoldWithMouseEnabled="True"/>
        </Control.GestureRecognizers>
```

Reply:
Then there will be two recognizers with different options for the same control

Reply:
@emmauss let's add an attached property instead.